guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -131,7 +131,7 @@ learn about Ruby on Rails, and the API, which serves as a reference.
 You can help improve the Rails guides by making them more coherent, consistent or readable, adding missing information, correcting factual errors, fixing typos, or bringing them up to date with the latest edge Rails.
 
 You can either open a pull request to [Rails](https://github.com/rails/rails) or
-ask the [Rails core team](http://rubyonrails.org/core) for commit access on
+ask the [Rails core team](http://rubyonrails.org/community) for commit access on
```

Comment:
http://rubyonrails.org/community/#core is a better link, autoscrolls to the core section on the page :)